### PR TITLE
Add more useful documents and issue tempaltes

### PR DESCRIPTION
**Pull request by JezSonic**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+## Request for changes/ Pull Requests
+You first need to create a fork of this repository to commit your changes to it. Methods to fork a repository can be found in the [GitHub Documentation](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+
+## Choose a base branch
+Before starting development, you need to know which branch to base your modifications/additions on. When in doubt, use master.
+
+| Type of change                |           | Branches              |
+| :------------------           |:---------:| ---------------------:|
+| Documentation                 |           | `master`              |
```
Comment (marcinn) [inline CONTRIBUTING.md:11]: master or main?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+| Bug fixes                     |           | `master`              |
+| New features                  |           | `master`              |
+| New issues models             |           | `master` |
+
+```sh
+# Switch to the desired branch
+git switch master
```
Comment (marcinn) [inline CONTRIBUTING.md:18]: main

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# Pull down any upstream changes
+git pull
+
+# Create a new branch to work on
+git switch --create dev/[feature-or-fix-name]
```
Comment (marcinn) [inline CONTRIBUTING.md:24]: not sure about the `dev` prefix requirement. everything is a dev.
consider bugfix/ feature/ release/

Comment (JezSonic) [reply CONTRIBUTING.md:24]: thx

File: CONTRIBUTING.md (hunk 2 above)
Comment (JezSonic) [reply CONTRIBUTING.md:18]: Thx

**Review by marcinn — APPROVED** [review]

**Pull request merged**